lib/tasks/email_checkin_reminder.rake

```diff
@@ -6,7 +6,8 @@ task email_checkin_reminder: :environment do
     Rails.logger.info "Found #{upcoming_reservations.size} reservations due "\
                       'for check-in. Sending reminder emails...'
     upcoming_reservations.each do |upcoming_reservation|
-      UserMailer.reservation_status_update(upcoming_reservation).deliver
+      m = UserMailer.reservation_status_update(upcoming_reservation).deliver
```

Comment:
I think we can move this into the `reservation_status_update` method so it doesn't have to be duplicated in all of the rake tasks; the only exception is in the `email_notes_to_admins` task which uses a separate method.